### DISCUSSION TITLE
windows 10 action center for notifications

### DISCUSSION THREAD

**Discussion by Rezy774** [valinet/ExplorerPatcher#2423] [Ideas]
i hate win 11 notifs and also how do i get sharp corners on seach menu

**Comment by pyrates999** [comment]
You can disable rounded corners:

![image](https://github.com/valinet/ExplorerPatcher/assets/106135298/e8d30fde-d31d-4e33-b581-252d3acbf008)

And for windows 11 notifications, right click on the notification icon and set focus assist to alarms only.

**Reply by Rezy774** [reply]
ty
